fission/function.go

```diff
@@ -207,12 +212,14 @@ func fnCreate(c *cli.Context) error {
 	}
 
 	invokeStrategy := getInvokeStrategy(c.Int("minscale"), c.Int("maxscale"), c.String("executortype"), getTargetCPU(c))
-	resourceReq := getResourceReq(c, apiv1.ResourceRequirements{})
+
 	if (c.IsSet("mincpu") || c.IsSet("maxcpu") || c.IsSet("minmemory") || c.IsSet("maxmemory")) &&
 		invokeStrategy.ExecutionStrategy.ExecutorType == fission.ExecutorTypePoolmgr {
-		log.Warn("CPU/Memory specified for function with pool manager executor will be ignored in favor of resources specified at environment")
+		log.Fatal("To limit CPU/Memory for function with executor type \"poolmgr\", please specify resources limits when creating environment")
```

Comment:
I think a warning is better here.  We can still execute the function, we just can't provide the requested resources.  Also, in the future if K8s supports changing resources on existing pods, we could potentially support this with poolmgr.